assets/js/polyfills.js

```diff
@@ -0,0 +1,8 @@
+import 'core-js/stable';
+import 'regenerator-runtime/runtime';
+import 'whatwg-fetch';
```

Comment:
Polyfilling `fetch` seems sensible while we're in here; allows us to start using it.